tests/test_edgeql_globals.py

```diff
@@ -404,6 +404,20 @@ async def test_edgeql_globals_12(self):
             [16]
         )
 
+    async def test_edgeql_globals_13(self):
```

Comment:
Could we add a test with config commands too? I seem to recall we might try to use the static evaluator on those so it's worth seeing what happens

Reply:
I'm not sure if this is possible. The non-global "config set" commands we have are:
- `set module mod;`
- `set alias al as module mod;`

https://www.edgedb.com/docs/reference/edgeql/sess_set_alias#set

Reply:
@aljazerzen https://www.edgedb.com/docs/reference/admin/configure#configure